addons/mail/static/src/core_ui/message.js

```diff
@@ -560,4 +585,25 @@ export class Message extends Component {
             message: this.props.message,
         });
     }
+
+    createShadowBody() {
+        if (this.message.type !== "email") {
+            return [];
+        }
+        const color = this.env.services.cookie?.current.color_scheme === "dark" ? "white" : "black";
+        const styleEl = document.createElement("style");
+        styleEl.innerHTML = `
+            * {
+                background-color: transparent !important;
+                color: ${color} !important;
+            }
+            a, a * {
+                color: blue !important;
+            }
+        `;
+        const messageEl = document.createElement("span");
+        messageEl.innerHTML = this.message.body;
+        this.insertReadMoreLess($(messageEl));
```

Comment:
Could be done in `onMounted()` so it's next to equivalent code with `.messageBody.el`